Geometry/TrackerGeometryBuilder/interface/TrackerGeometry.h

```diff
@@ -40,7 +40,8 @@ class TrackerGeometry final : public TrackingGeometry {
 
 public:
   typedef GeomDetEnumerators::SubDetector SubDetector;
-
+  enum ModuleType {UNKNOWN, PXB, PXF, IB1, IB2, OB1, OB2, W1A, W2A, W3A, W1B, W2B, W3B, W4, W5, W6, W7, Ph1PXB, Ph1PXF, Ph2PXB, Ph2PXF, Ph2PSP, Ph2PSS, Ph2SS};
```

Comment:
@suchandradutta - a strongly typed enum would be better here - e.g. a better scope. Please, use forward declaration and put specific values into the cc file.